File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacPersistance/DAO/PropuestasPagoDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface PropuestasPagoDAO
    {
        int Insertar(PropuestasPagoDTO propuestaPago);
        int Modificar(PropuestasPagoDTO propuestaPago);
        int Eliminar(PropuestasPagoDTO propuestaPago);
        int EliminarLogico(PropuestasPagoDTO propuestaPago);
        PropuestasPagoDTO ObtenerPorId(int propuestaPagoId);
        IList<PropuestasPagoDTO> ListarTodos();
        IList<PropuestasPagoDTO> ListarTodaActividad(); // <-- AÑADIR ESTA LÍNEA

    }
}

File: Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/PropuestasPagoDAO.java
package pe.edu.pucp.softpac.dao;

import java.util.List;
import pe.edu.pucp.softpac.model.PropuestasPagoDTO;

public interface PropuestasPagoDAO {
    public Integer insertar(PropuestasPagoDTO propuestaPago);
    
    public PropuestasPagoDTO obtenerPorId(Integer propuestaPagoId);
    
    public List<PropuestasPagoDTO> listarTodos();
        
    public Integer modificar(PropuestasPagoDTO propuestaPago);
    
    public Integer eliminar(PropuestasPagoDTO propuestaPago);

    public Integer eliminarLogico(PropuestasPagoDTO propuestaPago);
}
